amg/test/mltest/mltest.cpp

```diff
@@ -414,6 +414,10 @@ int main(int argc, char *argv[])
     args.AddOption(&identity_partition, "-ip", "--identity-partition",
                    "-nip", "--no-identity-partition",
                    "No agglomeration, one element per agglomerate.");
+    bool adapt = false;
+    args.AddOption(&adapt, "-ad", "--adapt",
```

Comment:
How hard would it be to add a test for this option? (I understand our test suite in this project is *very* incomplete, but if it is easy to add a quick test now is the time to do it.)

Reply:
Done